src/backend/access/transam/slru.c

```diff
@@ -764,19 +764,29 @@ SimpleLruDownloadSegment(SlruCtl ctl, int pageno, char const* path)
 
 	if (neon_use_communicator_worker) {
 		buffer = NULL;
+		pgstat_report_wait_start(WAIT_EVENT_SLRU_WRITE);
 	} else {
 		buffer = palloc(BLCKSZ * SLRU_PAGES_PER_SEGMENT);
 	}
```

Comment:
PG uses BSD-style bracing:

```suggestion
	if (neon_use_communicator_worker)
	{
		buffer = NULL;
		pgstat_report_wait_start(WAIT_EVENT_SLRU_WRITE);
	}
	else
	{
		buffer = palloc(BLCKSZ * SLRU_PAGES_PER_SEGMENT);
	}
```